build/apple/download_swift_format.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env vpython3
```

Comment:
is vpython3 necessary here? It looks like we use everything from the standard python3 distribution.

Reply:
Nope! good call, swapped with python3